### PR TITLE
add destroy method

### PR DESCRIPTION
**Pull request by souporserious**
Adds destroy method to allow cleanup after live region is no longer needed. Closes https://github.com/schne324/live-region/issues/1.

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -28,4 +28,10 @@ describe('LiveRegion', function () {
     });
   });
 
+  describe('destroy', function () {
+    it('should remove region node from the DOM', function (done) {
```
Comment (schne324) [inline test/index.js]: this `done` parameter is only needed for asynchronous tests (so its neither used nor needed here). I'd also suggest something a little more clear as a check if the element has been removed from the dom. Something like `assert.false(document.contains(polite.region);`

**Comment by schne324** [comment]
would you mind updating the readme by documenting this within the "Methods" section? (right below the [announce](https://github.com/schne324/live-region#liveregionannounce) method docs)

File: test/index.js (hunk 1 above)
Comment (souporserious) [reply test/index.js]: Oops, I copied and pasted from the one above 😇. And way better check! I was curious the best way to test that. Love using `contains` 🤘.

**Comment by schne324** [comment]
looks good, thanks @souporserious !

**Comment by souporserious** [comment]
Thank you!!! 🙏🎉

**Pull request merged**